charts/langsmith/values.yaml

```diff
@@ -388,6 +390,8 @@ clickhouse:
     # If enabled, use the following values to connect to an external database. This will also disable the
     # creation of a clickhouse stateful-set and service.
     enabled: false
+    # -- Either "self-hosted" or "hybrid"
+    hosting: "self-hosted"
```

Comment:
why not just boolean here? hybrid: true or false?

Reply:
had been thinking we would set to `cloud` and potentially add as env var, but yeah I don't think that's necessary. will update